src/components/wallet/create/ConfirmPassphrase.tsx

```diff
@@ -77,9 +58,42 @@ const ConfirmPassphrase = ({ goToNextStep, formik }: Props) => {
         evt: React.ChangeEvent<HTMLInputElement>,
         index: number,
     ) => {
-        formik.setFieldValue(`confirmPassphrase[${index}]`, evt.target.value);
+        const confirmPassphrase = values.confirmPassphrase;
+        confirmPassphrase[index] = evt.target.value;
+
+        formik.setFieldValue(`confirmPassphrase`, confirmPassphrase);
+
+        browser.runtime.sendMessage({
+            type: 'SET_LAST_SCREEN',
```

Comment:
on any other pages we should reset the last screen. Currently they are persisted even when you navigate away, e.g.:
- settings > create & import > create new address > go to passphrase screen
- navigate back to the main page
- close extension
- reopen extension, you're back on the passphrase screen

Reply:
it would be good if we have some automatic clearing between navigation, so each time you visit a page it first clears any persistent screen it may have and stores a new version afterwards if it needs that